modules/compute/aws-batch/app.py

```diff
@@ -12,6 +12,7 @@
 project_name = os.getenv("SEEDFARMER_PROJECT_NAME", "")
 deployment_name = os.getenv("SEEDFARMER_DEPLOYMENT_NAME", "")
 module_name = os.getenv("SEEDFARMER_MODULE_NAME", "")
+partition = os.getenv("AWS_PARTITION", "")
```

Comment:
Can we get the partition from the scope `App()` construct here? Or at least have a reasonable default?

Reply:
Sure. I can set the default value as `aws`. Since seedfarmer provides the partition as an env var, i was using it. 
Ref: https://github.com/awslabs/seed-farmer/blob/274e86ff617a9f4247e208dbfb2841843165f7ab/seedfarmer/commands/_module_commands.py#L84C15-L84C28

Reply:
Gotcha, sounds good to me